### DISCUSSION TITLE
Status of Meshcentral and Meshcommander

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5818] [General]
Hi, i've a nontechnical Question:
I viewed some videos at youtube that describes meshcentral an meshcommander. 
I'm trying to evaluate the Software and until now it seems, that many feature will fit to our needs. 

But the biggest problem for is, that i don't see what's the status about meshcentral an commander?
at meshcommander.com i see this here:

MeshCommander & More
Intel has ended support for these tools. Please contact Intel support for alternatives.

So what does this mean to Meshcommander and for meshcentral? IS it possible that meshcentral will be shutdown in six month or something like that? And what is the part from intel for this, that i have to look for alternatives?
I'm a bit worried about this. 
Regards Sebastian

**Comment by si458** [comment]
Both softwares are opensource, so they will live on with forks etc if they ever stop getting developed

Meshcommander is really only needed for intel amt things,
Where as meshcentral is a remote support software to control all devices

Features and bug fixes are still being worked on just at a slower pase

As for the intel support, basically intel paid/helped @Ylianst develop all the intel amt side of things, but as that support stopped, if the is any new features for amt, @Ylianst won't be able to integrate them without support from the community

@Ylianst anything extra you would like to add?

**Comment by tomsik-radek** [comment]
MeshCentral is an open source self hosted app. While yes, Intel stopped funding, security patches are still coming every few weeks. MeshCommander is a utility for Intel AMT.

What was shut down is the public meshcentral.com instance.

**Comment by marclaporte** [comment]
@simpsonetti and everyone:

Please get involved and make MeshCentral better! https://github.com/Ylianst/MeshCentral/discussions/5540
